transforms/angle-brackets/test.js

```diff
@@ -34,21 +36,13 @@ test('actions', () => {
   `;
 
   expect(runTest('actions.hbs', input)).toMatchInlineSnapshot(`
-    "<BsButtonGroup
-      @value={{buttonGroupValue}}
-      @type=\\"checkbox\\"
-      @onChange={{action (mut buttonGroupValue)}} as |bg|
-    >
-      <bg.button @value={{1}}>
-        1
-      </bg.button>
-      <bg.button @value={{2}}>
-        2
-      </bg.button>
-      <bg.button @value={{3}}>
-        3
-      </bg.button>
-    </BsButtonGroup>"
+    "
+        <BsButtonGroup @value={{buttonGroupValue}} @type=\\"checkbox\\" @onChange={{action (mut buttonGroupValue)}} as |bg|>
```

Comment:
Can you open an issue specifically about coordinating with ember-template-recast to create an API to allow a transform to customize whitespace for newly created nodes (in the case of this test, the whitespace cannot be inferred because its a fully new ElementNode) then leave an inline comment with a link to that issue?

I'll mention linking to the "new node whitespace preservation issue" in a few other spots as I notice them.

Reply:
This issue https://github.com/ember-template-lint/ember-template-recast/issues/82 has us discussing the above.  Can I use this issue for the comment or would you like a new one created?

Reply:
Ya, that seems great to link to. I mostly wanted to notate inline that we intend for this output to be changed in the future (once the issue has been fixed/released).